assets/sass/elements/_forms.scss

```diff
@@ -20,6 +20,11 @@ fieldset {
   width: 100%;
 }
 
+// Hack to let legends or elements within legends have margins in webkit browsers
```

Comment:
Ah, yes that works as well. I could have sworn I tried it but apparently not. I think that makes more sense, so will change it.